io.c

```diff
@@ -14700,47 +14700,286 @@ set_LAST_READ_LINE(VALUE val, ID _x, VALUE *_y)
 /*
  * Document-class:  ARGF
  *
- * ARGF is a stream designed for use in scripts that process files given as
- * command-line arguments or passed in via STDIN.
+ * :markup: markdown
  *
- * The arguments passed to your script are stored in the +ARGV+ Array, one
- * argument per element. ARGF assumes that any arguments that aren't
- * filenames have been removed from +ARGV+. For example:
+ * ## \ARGF and `ARGV`
  *
- *     $ ruby argf.rb --verbose file1 file2
+ * The \ARGF object works with the array at global variable `ARGV`:
  *
- *     ARGV  #=> ["--verbose", "file1", "file2"]
- *     option = ARGV.shift #=> "--verbose"
- *     ARGV  #=> ["file1", "file2"]
+ * - **ARGV** may be thought of as the <b>argument vector</b> array.
  *
- * You can now use ARGF to work with a concatenation of each of these named
- * files. For instance, ARGF.read will return the contents of _file1_
- * followed by the contents of _file2_.
+ *     Initially, it contains the arguments and options
+ *     that are passed to it from the command line;
+ *     the running Ruby program can modify that array as it likes.
```

Comment:
What do "options" mean here?
It sounds like options to ruby interpreter itself.

Reply:
Rephrased.